core/src/main/scala/com/salesforce/op/evaluators/OpMultiClassificationEvaluator.scala

```diff
@@ -188,7 +188,8 @@ private[op] class OpMultiClassificationEvaluator
     def computeMetrics(scoresAndLabels: (Array[Double], Double)): MetricsMap = {
       val scores: Array[Double] = scoresAndLabels._1
       val label: Label = scoresAndLabels._2.toInt
-      val trueClassScore: Double = scores(label)
+      // The label may be unseen during model training, so treat scores for unseen classes as all being zero
+      val trueClassScore: Double = if (scores.isDefinedAt(label)) scores(label) else 0.0
```

Comment:
how about `scores.lift(label).getOrElse(0.0)`

Reply:
Ahh, I was trying to remember how to do that!